.travis.yml

```diff
@@ -55,14 +56,16 @@ jobs:
         export JAVA_HOME=/usr/lib/jvm/adoptopenjdk-8-hotspot-amd64
         export PATH=/usr/lib/jvm/adoptopenjdk-8-hotspot-amd64/bin:$PATH
         java -version && docker-compose up -d cassandra && sbt -jvm-opts .jvmopts-travis +test
-      name: Run all tests with Jdk 8
+      name: Test / JDK 8 / Cassandra Latest
     - script: java -version && docker-compose up -d cassandra && sbt -jvm-opts .jvmopts-travis +test
-      name: Run all tests with Jdk 11
+      name: Test / JDK 11 / Cassandra Latest
```

Comment:
We can look into if we can define `cassandra.yaml` and enable materialized views that way (as stated in the error message). Otherwise I'd be fine with not running the migration tests with Cassandra 4.

Reply:
I was inclined to tag the failing tests with `RequiresCassandraThree` and skip them when running against Cassandra 4.

Reply:
Although I'm pretty sure we can enable materialized views using `cassandra.yaml`, but I don't think there is anyone using old versions of APC _and_ migrating to Cassandra 4.

Reply:
Sounds good to skip these test for 4

Reply:
@patriknw I've decided instead to define a `cassandra.yaml` configuration for `cassandra:latest`. It was better that way to get another small problem in the tests.